data/model/user.py

```diff
@@ -111,6 +111,34 @@ def create_user(
     return created
 
 
+def create_init_user(
+    username,
+    password,
+    email,
+):
+    """
+    Creates an initial user, if none are in the database.
+    """
+
+    if config.app_config.get("AUTHENTICATION_TYPE") != "Database":
+        logger.debug("Cannot create init user in a non-database auth system")
+        return
+
+    if bool(list(User.select().limit(1))):
```

Comment:
We've generally favored running the query and catching the "DoesNotExist" exception when testing for existence.
e.g 
```
try:
  User.select().limit(1).get()
except UserDoesNotExist:
  logger.debug("Cannot initialize user in a non-empty database")
  return
```